lib/features/children/edit_child/repositories/edit_child_repository.dart

```diff
@@ -65,4 +68,12 @@ class EditChildRepositoryImpl with EditChildRepository {
 
   @override
   Stream<String> childChangedStream() => _childGUIDController.stream;
+
+  @override
+  Future<bool> editChildBedtime(Bedtime bedtime) async {
+    final response = await apiService.editChildBedtime(
+        bedtime.id, bedtime.bedtimeInUtc, bedtime.winddownMinutes);
+    _addChildGuidEventAfterOneSecond(bedtime.id);
```

Comment:
The afterOneSecond was mainly used for anything payment related. We don't need to wait a second with this one unless you observed that the delay was needed.